src/Reduce.php

```diff
@@ -175,4 +175,32 @@ public static function isReversed(iterable $data): bool
 
         return true;
     }
+
+    /**
+     * Returns true if given collection is exactly the same as another ones.
+     *
+     * @param iterable<mixed> $data
+     * @param iterable<mixed> $n
+     * @param iterable<mixed> ...$extra
+     *
+     * @return bool
+     */
+    public static function exactlyN(iterable $data, iterable $n, iterable ...$extra): bool
+    {
+        $iterables = [$data, $n, ...$extra];
```

Comment:
Question: Are the three different parameters needed? It seems like you can just have the method declaration be a single variadic parameter like this:
```php
public static function exactlyN(iterable ...$iterables): bool
```

And then pass that to zip and not need the extra variable assignment here.

Reply:
Done.